packages/ethereum-contracts/contracts/interfaces/agreements/IConstantFlowAgreementV1.sol

```diff
@@ -143,6 +143,42 @@ abstract contract IConstantFlowAgreementV1 is ISuperAgreement {
         bytes calldata ctx
     ) external virtual returns(bytes memory newCtx);
 
+    /**
+     * @dev msgSender from `ctx` increases flow rate allowance for the `flowOperator` by `addedFlowRateAllowance`
+     * @dev if `addedFlowRateAllowance` is negative, we revert with CFA_ACL_NO_NEGATIVE_ALLOWANCE
+     * @param token Super token address
+     * @param flowOperator The permission grantee address
+     * @param permissionsToAdd A bitmask representation of the granted permissions to add as a delta
+     * @param addedFlowRateAllowance The flow rate allowance delta
+     * @param ctx Context bytes (see ISuperfluid.sol for Context struct)
+     * @return newCtx The new context bytes
+     */
+    function increaseFlowAllowanceWithPermissions(
```

Comment:
It's a tiny detail but there is a bit of a new term "flow allowance" used here as opposed to "flow rate allowance".

Fun fact, in the products, we've been calling it "stream allowance".

Reply:
Yea, some other functions are named "increaseFlow**Rate**Allowance" and "decreaseFlow**Rate**Allowance".

Reply:
This was done consciously, mainly due to verbosity of the function name, but I see your point here, it is inconsistent and may lead one to question if there is a difference in behavior between the two functions.